tests/support/mod.rs

```diff
@@ -1,5 +1,11 @@
 //! Utilities to support tests.
 
+#[cfg(not(feature = "unstable"))]
+compile_error!(
```

Comment:
If you forget to enable the `unstable` feature when running the tests, the error message is quite bizarre (about duplicate functions on `frames::Mock` and so forth). We can make it super clear with this.